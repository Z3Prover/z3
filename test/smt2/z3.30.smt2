
; Copyright (c) 2015 Microsoft Corporation
(set-option :auto-config true)
(set-option :produce-models true)

(set-option :print-success true) 
(set-option :produce-unsat-cores true) ; enable generation of unsat cores
(set-option :produce-models true) ; enable model generation
(set-option :smt.mbqi true) ; enable model based quantifier instantiation
(set-option :produce-proofs true) ; enable proof generation
(declare-const x Int)
(set-option :produce-proofs false) ; error, cannot change this option after a declaration or assertion
(echo "before reset")
(reset)
(set-option :produce-proofs false) ; ok


