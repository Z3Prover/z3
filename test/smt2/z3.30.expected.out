success
success
success
success
success
success
(error "line 12 column 28: error setting ':produce-proofs', option value cannot be modified after initialization")
before reset
success
success
